memsetVsMadvise.cpp

```diff
@@ -80,14 +115,14 @@ int main(int argc, char** argv) {
     "This program benchmarks memset vs madvise for zeroing memory.\n"
     "Sample usage:\n";
   usage += argv[0];
-  usage += " --max_pages=20 --num_runs=30 --num_threads=4 ";
-  usage += "--touch_after_zero=true";
+  usage += " --num_pages_min=20 --num_pagse_max=50 --num_runs=30 ";
```

Comment:
I guess we can evaluate even larger sizes? It probably benefits most with larger buffers.